lib/components/atoms/Image/tests/Image.test.js

```diff
@@ -12,7 +13,7 @@ describe('Image Component', () => {
 
   test('should render correctly and match snapshots', () => {
     const image = shallow(<ImageVanilla {...sampleProps} />);
-    expect(image).toMatchSnapshot();
+    expect(tojson(image)).toMatchSnapshot();
```

Comment:
Why?

Reply:
So that we can identify is there any breakage of code while adding the other functionality.

Reply:
Please revert this, its no longer needed.

Reply:
done